src/libstore/include/nix/store/store-registration.hh

```diff
@@ -0,0 +1,94 @@
+#pragma once
+/**
+ * @file
+ *
+ * Infrastructure for "registering" store implementations. Used by the
+ * store implementation definitions themselves but not by consumers of
+ * those implementations.
```

Comment:
"not" doesn't tell me much. Where should they look instead?

Reply:
Extended with more info